tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "typestrict",
+  "rules": {
+    "no-console": true,
+    "no-debugger": true,
```

Comment:
To make sure: this doesn't prevent on every day development from saving and locally using ``console`` or ``debug`` statements, does it?

Reply:
Soooo, I never run linting constantly while developing. When I am about to commit I do `yarn test:fix` and I am ready to go.